crates/bevy_reflect/src/lib.rs

```diff
@@ -55,6 +55,34 @@ pub use type_info::*;
 pub use type_registry::*;
 pub use type_uuid::*;
 
+/// A catch-all trait bound by the core reflection traits for easy reflection-based trait bounds.
+///
+/// > __Note:__ You do _not_ need to implement this trait manually.
```

Comment:
```suggestion
/// You do _not_ need to implement this trait manually.
```

No need to go crazy on the emphasis :)

Reply:
Yeah thanks for reeling me back haha I tend to go overboard 😅